angular/src/app/screens/sign-up/sign-up.module.ts

```diff
@@ -8,11 +8,6 @@ import { FormContainerComponent } from 'src/app/components/form-container/form-c
 @NgModule({
   declarations: [SignUpComponent, FormContainerComponent],
   imports: [CommonModule, SignUpRoutingModule, ReactiveFormsModule],
-  exports: [
-    SignUpComponent,
-    FormContainerComponent,
-    CommonModule,
-    ReactiveFormsModule,
-  ],
+  exports: [SignUpComponent, FormContainerComponent, CommonModule],
```

Comment:
It's not necessary to export components if you not using them in an external module